src/views/DocumentationTopic.vue

```diff
@@ -477,6 +481,10 @@ export default {
   }
 }
 
+.background-disabled {
```

Comment:
This change can likely be avoided altogether if we instead simply override the `--color(s)-text-background` variable to be transparent for IDE targets.